cache-redis-cache-arm-provision/readme.md

```diff
@@ -0,0 +1,4 @@
+# Create a Redis Cache using a template
```

Comment:
It might be better to add a 101 to this template name since it seems pretty basic. A better name would be 101-redis-cache